### ISSUE TITLE
[Question] Is it compatible with Neato Botvac Connected?

### ISSUE THREAD

**Issue by vitorrm** [RobertSundling/neato-botvac#3]
Hi,

Do you happen to know if this firmware will also work on the "Neato Botvac Connected" (Non D)? If not, do you have any clue where I can find the latest firmware for it?

amazon.com/Neato-Botvac-Connected-Enabled-Vacuum/dp/B0168KHVZW

I can't get it to connect to the servers for a long time now.

Thanks

**Comment by RobertSundling** [comment]
This firmware is almost certainly incompatible. The D series Botvacs dropped the LCD display and the buttons around that, and are quite different than the original Connected.

Unfortunately I don’t know where to get the firmware for the original Connected. I do actually happen to own one of those, too, but since it’s been working I don’t want to experiment with it. However, if it has problems at some point in the future and I need to replace the firmware on it, I may go through a similar process as I did with my D7 Connected and figure out how to get that firmware as well.

I might suggest asking on reddit or on the [Robot Reviews forum](http://www.robotreviews.com/chat/) which is where I found a lot of information.
